helpers/calendar/all_analytics_helper.py

```diff
@@ -37,7 +45,9 @@ def map_bookings_to_period(dates, date_pattern, events):
         bookings_count = []
         for date in dates:
             string_date = dateutil.parser.parse(date[0]).strftime(date_pattern)
-            bookings = AllAnalyticsHelper.count_bookings_within_period(events, date_pattern, string_date) # noqa
+            bookings = AllAnalyticsHelper.count_bookings_within_period(
+                events, date_pattern, string_date
+            )  # noqa
```

Comment:
You don't need the "# noqa" at the end of this line.

Reply:
Done. Thank you for the feedback @adafia